### PR TITLE
Cad dash mock

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

- Adds Canadian Dashboard Tableau prototype (viewable at the /Canada URL, no visual entry point from the main site for now)
- Fixes double scrollbar issues
- Refactors page routing

## Please link the Airtable ticket associated with this PR.

- https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recz3tBozneEUmRVS 

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).

image

image

## Describe the steps you took to test the feature/bugfix introduced by this PR.

- Visited /Canada and /Analyze tabs and ensured that tableau vizzes looked good & contained the right content
- General regression testing (click through all pages, try switching between pages, etc)

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.

No.

### CODE REVIEW

**Comment by atatmaja** [comment]
@brett-stephen regarding your question on the last pr, the tableau embed heights are defined in the tableau UI (I think bc they use SSR). The height prop that I'm supplying only affects the container around that embed.

For that reason, I think that on monitors of diff resolutions it should be ok

File: src/App.tsx [hunk 1]
```diff
@@ -16,7 +16,31 @@ import httpClient from "./httpClient";
...
+  )
+}
+
+function AnalyzeTableau() {
+  return (
+    <TableauEmbed
+      url={ANALYZE_URL}
+      options={{
+        width: "80vw",
+        height: "3200px"
```
Comment (brett-stephen) [inline src/App.tsx:39]: have these sizes been tested on mobile?

**Review by brett-stephen — APPROVED** [review]
looks good

File: src/App.tsx (hunk 1 above)
Comment (atatmaja) [reply src/App.tsx:39]: They look okay ish, but I think the precise heights/widths for mobile rendering are also defined in tableau so I will sync with jordan on that and push changes accordingly

**Pull request merged**